dvc/repo/dvclive.py

```diff
@@ -0,0 +1,52 @@
+import contextlib
+import logging
+import os
+from typing import List
+
+from dvc.exceptions import NoMetricsError
+from dvc.output import BaseOutput
+from dvc.path_info import PathInfo
+from dvc.repo import Repo
+from dvc.visualization import embed, metrics_embedding, plots_embeddings
+
+logger = logging.getLogger(__name__)
+
+
+def summary_path_info(out: BaseOutput) -> PathInfo:
+    assert out.dvclive
+    has_summary = True
+    if isinstance(out.dvclive, dict):
+        has_summary = out.dvclive.get(BaseOutput.PARAM_DVCLIVE_SUMMARY, True)
+    if has_summary:
+        return out.path_info.with_suffix(".json")
+    return None
+
+
+class DvcLive:
+    def __init__(self, repo: Repo):
+        self.repo = repo
+
+    def summarize(self, path: str, revs: List[str] = None):
```

Comment:
Maybe let's do `summarize` -> `show` so it matches the other commands (metrics, plots, etc)? As I understand it is not user facing. Also `dvclive,py` -> `live.py`

Reply:
done, also adjusted returned values to match `metrics` / `plots`